provider/oci/oci.go

```diff
@@ -96,7 +96,7 @@ func NewOCIProvider(cfg OCIConfig, domainFilter endpoint.DomainFilter, zoneIDFil
 		&cfg.Auth.Passphrase,
 	))
 	if err != nil {
-		return nil, errors.Wrap(err, "initialising OCI DNS API client")
+		return nil, errors.Wrap(err, "initializing OCI DNS API client")
```

Comment:
`initialising` is not a typo :).

Unless we recognise only AmE though.

Reply:
Yeah agree but I would like following AmE instead of BE